docs/internals/contributing/writing-documentation.txt

```diff
@@ -256,6 +256,25 @@ If a function, attribute, etc. is added, it's also okay to use a
 We can simply remove the ``.. versionadded:: A.B`` annotation without any
 indentation changes when the time comes.
 
+Minimizing images
+-----------------
+
+Optimize image compression where possible.  For PNG files, try OptiPNG and AdvanceCOMP's ``advpng``:
+
+.. code-block:: sh
+
+   $ cd docs/
+   $ optipng -o7 -zm1-9 -i0 -strip all `find . -type f -name "*.png"`
+   $ advpng -z4 `find . -type f -name "*.png"`
+
+If you have CPU power and time to spare, you can use the `-i` option with ``advpng``:
+
+.. code-block:: sh
+
+   $ advpng -z4 -i 1000 `find . -type f -name "*.png"`
```

Comment:
It might be better to omit this. I'd like to standardize on what command to run each time so that each time we add or modify an image, we can run the optimization command without generating change images for all the other images... if this seems like a reasonable goal?